engine/flink/kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/generic/FlinkKafkaDelayedSourceImplFactory.scala

```diff
@@ -132,10 +132,10 @@ class FlinkKafkaDelayedSourceImplFactory[K, V](
         DelayedFlinkKafkaConsumer(
           preparedTopics,
           deserializationSchema,
-          kafkaConfig,
+          this.kafkaConfig,
```

Comment:
why do we need that?

Reply:
Because of a compiler warning (which we treat as a compilation error): in Scala 3 "kafkaConfig" will refer to "kafkaConfig" from outer scope (line 110), and not to "kafkaConfig" which we pass to constructor in line 120.